shopfloor/actions/data.py

```diff
@@ -11,13 +11,37 @@ class DataAction(Component):
 
     @ensure_model("stock.location")
     def location(self, record, **kw):
```

Comment:
```suggestion
    def location(self, record, with_operations_progress_info=False, **kw):
```

Reply:
I'm following the kw approach because this is what we did for the pickigns data:

```    
def picking(self, record, **kw):
        parser = self._picking_parser
        # progress is a heavy computed field,
        # and it may reduce performance significatively
        # when dealing with a large number of pickings.
        # Thus, we make it optional.
        if "with_progress" in kw:
            parser.append("progress")
        return self._jsonify(record, parser, **kw)
```

We should try to keep it consistent, shouldn't we?

Reply:
Fair enough. In any case here we would just  "book" a specific key to be extracted from the generic kw.